edb/pgsql/parser/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.so
+psqlparse/parser.c
+build/
+__pycache__/
+*.c
```

Comment:
Make sure files have the trialing newlines; a lot of tools get irritated about it